enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -488,13 +488,52 @@ release-planning question.
 
 TBD.
 
+#### Automated and Manual Gateway Deployments
+
+OpenShift Service Mesh and Istio have a feature called [automated deployment](https://istio.io/latest/docs/tasks/traffic-management/ingress/gateway-api/#automated-deployment)
+that creates an Envoy deployment and service in the same namespace for each Gateway
+if the Gateway's spec.addresses field is left unset. This enables users to seamlessly
+create a "shard" per Gateway, much like how each IngressController object is a shard
+for routes. It is enabled via the `PILOT_ENABLE_GATEWAY_API_DEPLOYMENT_CONTROLLER`
+env variable.
+
+Conversely, with [manual deployments](https://istio.io/latest/docs/tasks/traffic-management/ingress/gateway-api/#manual-deployment),
+if the Gateway has the spec.addresses field set, then it must manually link
+to a service for a gateway deployment. The user needs to make their own gateway
+service and deployment in the same namespace to manually link to, or they need to use
+the default ingress gateway (if enabled).
+
+Istio, by default, creates a `istio-ingressgateway` service that is a ready-to-use
+gateway deployment which users can manually link to. It can be disabled
+via the IstioOperator spec.ingressGateways[].enabled field, but since OSSM Istio's Maistra
+fork does not currently expose these options, it is unable to be disabled.
+
+_To summarize, there are three methods a user could link Gateways:_
+1. Use Automated Deployments
+2. Use a Manual Deployment and manually create a new gateway service and deployment
+3. Use a Manual Deployment, but link to the default `istio-ingressgateway` service
+
+Openshift will not inhibit or alter the functionality of Automated Deployments, except
+restricting creation of Gateways to specific users (_see the Automated Deployment section
+in Risks and Mitigations_). The installation procedure (TBD) will allow users to
+enable/disable automated deployments.
+
+Nor will Openshift inhibit or alter the functionality of Manual Deployments. Users
+are responsible for understanding and creating links to manual deployments when creating
+Gateways.
+
+The choice between these two features depends on the user's desired sharding scheme. Manual
+linking, being more expressive, can establish a many-to-one Gateway-to-Gateway-Deployment
+relationship, while Automated Deployments strictly establish a one-to-one
+Gateway-to-Gateway-Deployment relationship. Arguably, Automated Deployments are more portable
+among Gateway API implementations due to the fact Manually Deployments require linking an
+Istio-specific service address.
```

Comment:
Can the end-user create a gateway using automated deployment and then create a second gateway configured to use the first gateway's service?